source/docs/robot-design/odometry.rst

```diff
@@ -9,25 +9,95 @@ Odometry
     designs and mechanisms with software during
     the off-season.
 
-Odometry is a form of localization that uses data
-from sensors like encoders. Localization is a
-means for being able to locate the position
-of the bot at some point in time. This is useful
-for `software <../software/odometry.html>`_, but in order
-to produce accurate results, teams need to design
-hardware that can be used as the sensors for localization.
+Odometry refers to the use of motion sensors for localization.
+Localization is a means for being able to locate the position
+of the bot at some point in time. Localization is crucial in
+path following and advanced autonomous modes as one needs to
+know where they are to know what movements they must make to
+reach a desired destination.
+`Localization software <../software/odometry.html>`_ plays, a
+major role in odometry; however, in order
+to produce accurate results, reliable and accurate hardware
+design is a necessity.
+
+The term dead wheels, tracking wheels, and odometry are often
+conflated in the FTC community. However, there are a
+few key differences one must keep in mind. Odometry refers to
+the general use of motion sensors for localization purposes.
+
+The simplest form of odometry is drive encoder localization.
+This is the use of encoders measuring the rotation of motors
+that power the drive train. One is able to read the encoder
+data and feed it through the kinematic equation for that
+specific drive train to derive the body's velocity. Drive
+encoder localization is generally quite simple and easy to setup as
+almost all of the FTC legal motors have built-in encoders. Getting
+drive encoder localization setup is simply a matter of plugging
+in wires, no additional hardware needed.
+
+Many teams in the community have converged on a unique solution that
+isn't seen very much outside of FTC: the use of "dead wheels,"
+"tracking wheels," or "odometry pods" (these terms are all synonymous).
+These refer to small "dead" or non-driven (not powered by a motor)
+wheels attached to an `encoder sensor <#encoders>`_. Two or three
+dead wheel pods are often sprung to the ground to ensure accurate
+tracking. The two-wheel design utilizes one parallel and one
+perpendicular pod (parallel and perpendicular with respect to the
+drive wheel axis), measuring x and y movement respectively. Change
+in heading is measured via a gyroscope. The three-wheel design
+utilizes two parallel and one perpendicular pod, measuring x and y
+movement respectively. However, this design forgoes the gyroscope
+and instead measures heading via the difference with the two parallel
+wheels. This is often more accurate in the context of the FTC control
+system because the BNO055 IMU (used for the gyroscope in the two-wheel
+design) utilizes i²c which is slower than the rest of the I/O
```

Comment:
```suggestion
design) utilizes I2C which is slower than the rest of the I/O
```

Reply:
The 2 is a squared tho

Reply:
Unicode sometimes fails on pdf builds it's p wack i just try to not include it